common/beerocks/hostapd/include/configuration.h

```diff
@@ -79,7 +97,7 @@ class Configuration {
     // { "wlan0.2" : ["ctrl_interface=/var/run/hosXXpd", "ap_isolate=1", "ap_max_inactivity=60", "bss_transition=0", "interworking=3"] }
     std::map<std::string, std::vector<std::string>> m_hostapd_config_vaps;
 
-    std::string m_error_description = "no error";
+    std::string m_last_message = "all good";
```

Comment:
```suggestion
    std::string m_last_message = "";
```
Use empty message for no error